packages/stories/src/gitgraph-js/6-custom-renders.stories.tsx

```diff
@@ -300,4 +306,235 @@ storiesOf("gitgraph-js/6. Custom renders", module)
+
+    // CustomRenderer class - note the vanilla JS syntax as it appears in storybook
+    // is "ugly" but modern browsers support the `class` keyword which provides
+    // prettier syntax. The source code for this story uses the ES6 class, which gets
+    // transpiled to the prototype-based syntax you see in storybook component explorer.
+    class MessageBelowTagsRenderer extends Renderer {
+
+      // copied the entire `positionCommitsElements` method with a tiny change
+      // see the lines marked with "** changed **" for the difference
+      positionCommitsElements() {
+        if (this.gitgraph.isHorizontal) {
+          // Elements don't appear on horizontal mode, yet.
+          return;
+        }
+
+        const padding = 10;
+
+        // Ensure commits elements (branch labels, message…) are well positionned.
+        // It can't be done at render time since elements size is dynamic.
+        Object.keys(this.commitsElements).forEach((commitHash) => {
+          const {branchLabel, tags, message} = this.commitsElements[commitHash];
+
+          // We'll store X position progressively and translate elements.
+          let x = this.commitMessagesX;
+
+          if (branchLabel) {
+            this.moveElement(branchLabel, x);
+
+            // BBox width misses box padding
+            // => they are set later, on branch label update.
+            // We would need to make branch label update happen before to solve it.
+            const branchLabelWidth =
+                branchLabel.getBBox().width + 2 * BRANCH_LABEL_PADDING_X;
+            x += branchLabelWidth + padding;
+          }
+          let highestTag = 0; // ** changed ** keeping track of highest tag
+          tags.forEach((tag) => {
+            this.moveElement(tag, x);
+
+            // BBox width misses box padding and offset
+            // => they are set later, on tag update.
+            // We would need to make tag update happen before to solve it.
+            const offset = parseFloat(tag.getAttribute("data-offset") || "0");
+            const tagWidth = tag.getBBox().width + 2 * TAG_PADDING_X + offset;
+            x += tagWidth + padding;
+
+            // ** changed ** - keeping track of highest tag
+            highestTag = Math.max(highestTag, tag.getBBox().height)
+          });
+
+          if (message) {
+            // ** changed ** - moving the message down instead of to the right
+            this.moveElementByXY(message, this.commitMessagesX, highestTag+padding+8);
+            // this.moveElement(message, x);
+          }
+        });
+      }
+    }
+
+    return (
+        <GraphContainer>
+          {(graphContainer) => {
+
+            const gitgraph = createGitgraph(graphContainer, {
+                  generateCommitHash: createFixedHashGenerator(),
+                },
+                () => {
+                  // keep reference to renderer so we can call re-render whenever `selectedCommit` changes
```

Comment:
This comment doesn't make much sense, I think this was a copy-paste mistake as we don't keep the reference here.

Reply:
Oops, You are right. I copied code from my project to this example and cleaned it up a bit, but forgot to remove this comment.